src/Forms/ChangeCasingNames.cs

```diff
@@ -230,10 +227,12 @@ internal void FixCasing()
             {
                 if (item.Checked)
                 {
-                    _noOfLinesChanged++;

```

Comment:
I think this is probably a 'bug', **_noOfLinesChanged** should only be incremented if **paragraph** is not **null**, right?